DMX.Sdk/Brokers/DmxApis/DmxApiBroker.cs

```diff
@@ -0,0 +1,38 @@
+﻿// --------------------------------------------------------------- 
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// ---------------------------------------------------------------
+
+using RESTFulSense.Clients;
+
+namespace DMX.Sdk.Brokers.DmxApis
+{
+    public partial class DmxApiBroker : IDmxApiBroker
+    {
+        private HttpClient httpClient;
+        private readonly string apiUrl;
+        private readonly IRESTFulApiFactoryClient apiClient;
+        private readonly string token;
+
+        public DmxApiBroker(HttpClient httpClient,
```

Comment:
I don't think we need to even inject `HttpClient` here. Just initialize it within this class.